src/controller/CHIPDeviceController.cpp

```diff
@@ -642,6 +658,16 @@ CHIP_ERROR DeviceCommissioner::EstablishPASEConnection(NodeId remoteDeviceId, Re
                                                   params.GetPeerAddress().GetInterface());
     }
 
+    if (FindCommissioneeDevice(peerAddress) != nullptr)
```

Comment:
So I'm trying to understand: what does the commissioner actually do here?  Is it expected to go through the entire "look up this device via advertising" process, until we land in here and coalesce with our existing open PASE session?  But how do we know this PASE session is in fact open?  That is, what is the sort of commissioning failure that keeps the PASE session open but leads to the commissioner thinking it should open a new one?